### ISSUE TITLE
[Visual Issue] Badge

### ISSUE THREAD

**Issue by kyranb** [livewire/flux#652]
Just a very minor hiccup, when setting a heading size, inset badges do not display in line with the same line height. 

Hopefully Caleb's AI picks this one up as an easy win ;)

The following displays the badge out of line with the heading text:

```
 <flux:heading size="xl" level="1">
    Heading Text <flux:badge color="lime" inset="top bottom">Badge Text</flux:badge>
 </flux:heading>
```

![Image](https://github.com/user-attachments/assets/fcb9694a-341e-4bff-9a9e-8ace7a44a538)

However, everything looks as expected when the `size=xl` is not set:

```
        <flux:heading level="1">
            Heading Text <flux:badge color="lime" inset="top bottom">Badge Text</flux:badge>
        </flux:heading>
```

![Image](https://github.com/user-attachments/assets/450009f9-88c8-4251-bb1f-6cb2a696c502)

**Comment by calebporzio** [comment]
Yeah, I would recommend using Flex box for the vertical centering you're looking for:

```
<flux:heading size="xl" level="1" class="flex items-center gap-2">
        Heading Text <flux:badge color="lime" inset="top bottom">Badge Text</flux:badge>
</flux:heading>
```

![Image](https://github.com/user-attachments/assets/1bf694c0-d192-4e1f-9eb2-e619298bbaa9)

The `inset` prop is meant to simply remove the extra height added by the extra padding (using negative margin) - not necessarily center your badge.
